sandbox/tests/test scenes/basis modifiers/30 - normal mapping - glossy brdf - area light.appleseed

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The `*.png` file in the `ref/` folder needs to have the same name as this `.appleseed` file, otherwise the testsuit won't recognize it.

Reply:
OK. Fixed now.